src/starbase/config.ts

```diff
@@ -34,16 +72,17 @@ async function loadRawConfig(filePath: string) {
   }
 }
 
-export async function loadParsedConfig(
+async function loadParsedConfig(
   filePath: string,
 ): Promise<StarbaseConfig> {
   const rawFile = await loadRawConfig(filePath);
   return yaml.load(rawFile) as StarbaseConfig;
 }
 
 async function validateStarbaseConfigSchema(config: StarbaseConfig) {
-  let finalConfig: StarbaseConfig = { integrations: [] };
+  const finalConfig: StarbaseConfig = { integrations: [] };
```

Comment:
Switched usage to `const` in various places. I thought we had a linting rule for preferring `const`. We should look into this...